### PR TITLE
feat(homepage): updated value props!

### PR DESCRIPTION
**Pull request by brooksmtownsend**
This PR updates some of our homepage value props to better reflect our goals as a project.

### CODE REVIEW

**Review by ricochet — APPROVED** [review]

File: docusaurus.config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 /** @type {import('@docusaurus/types').Config} */
 const config = {
   title: 'wasmCloud',
-  tagline: 'Develop. Deploy. Distribute.',
+  tagline: 'Build applications in any language and deploy them anywhere.',
```
Comment (connorsmith256) [inline docusaurus.config.js:10]: What do you think about "build distributed applications..."?

**Review by connorsmith256 — APPROVED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (brooksmtownsend) [reply docusaurus.config.js:10]: I do like that

Comment (ricochet) [reply docusaurus.config.js:10]: hrm, somewhat redundant with "deploy them anywhere"

Comment (connorsmith256) [reply docusaurus.config.js:10]: @ricochet I disagree. One technology could make it easy to deploy monoliths anywhere. Another could make it easy to deploy distributed apps, but only to x86 Linux. wasmCloud has both value props :)

**Pull request merged**